.github/workflows/techdocs.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Publish Backstage TechDocs'
+
+on:
+  workflow_dispatch:
+
+  # Run Daily at 10:00 UTC time
+  schedule:
+    - cron: '0 10 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-techdocs-site:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Backstage
+        uses: actions/checkout@v3
+        with:
+          repository: backstage/backstage
+          fetch-depth: 1
+
+      - name: use node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/ # Needed for auth
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install techdocs-cli
+        run: sudo npm install -g @techdocs/cli
+
+      - name: Install mkdocs and mkdocs plugins
+        run: python -m pip install mkdocs-techdocs-core==1.*
+
+      - name: Generate docs site
+        run: techdocs-cli generate --no-docker --verbose
+
+      - name: Publish docs site
+        run: techdocs-cli publish --publisher-type googleGcs --storage-name ${{ secrets.TECHDOCS_GCS_BUCKET }} --entity default/component/backstage
```

Comment:
Should there be a colon after the kind there?

Reply:
Yeah this is the expected format, not sure about the reasoning behind the choice of `namespace/kind/name` instead of the entity ref, but I think the `:` character is not recommended to use in storage object names. Could be one reason.

https://backstage.io/docs/features/techdocs/cli/#publish-generated-techdocs-sites